src/org/pentaho/platform/dataaccess/datasource/api/DataSourceWizardService.java

```diff
@@ -69,6 +70,9 @@
   protected IModelerService modelerService;
   protected IDatasourceMgmtService datasourceMgmtSvc;
 
+  protected IAclAwarePentahoMetadataDomainRepositoryImporter aclMetadataHelper;
```

Comment:
The "helper" portion of these names is confusing. Nothing huge, just clean it up next time you're in there please